lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.ThermalZone.rb

```diff
@@ -149,4 +149,22 @@ def thermal_zone_get_fan_power_limitations(thermal_zone, is_energy_recovery_requ
     # assuming that all supply air passes through all devices
     return fan_pwr_adjustment_in_wc * dsn_zone_air_flow_cfm / 4131
   end
+
+  # Identify if zone has district energy for occ_and_fuel_type method
+  # @param themal_zone
+  # @return [string] with applicable DistrictHeating and/or DistrictCooling 
+  def get_zone_fuels_for_occ_and_fuel_type(zone)
```

Comment:
To be consistent with other methods, the name should probably be `thermal_zone_get_district_heating_zones` since it's under ThermalZone.rb`